saleor/graphql/shipping/types.py

```diff
@@ -65,6 +65,13 @@ class Meta:
     def resolve_channel(root: models.ShippingMethodChannelListing, info):
         return ChannelByIdLoader(info.context).load(root.channel_id)
 
+    @staticmethod
+    def resolve_minimum_order_price(root: models.ShippingMethodChannelListing, info):
```

Comment:
What about `maxium_order_price`?

Reply:
Also, we need to make sure to support that in `ShippingMethodType.minimumOrderPrice` and `ShippingMethodType.maxiumOrderPrice`?

Reply:
> What about `maxium_order_price`?
`maxium_order_price_amount` doesn't have default value so `null` value works ok for `maxium_order_price`

Reply:
> Also, we need to make sure to support that in `ShippingMethodType.minimumOrderPrice` and `ShippingMethodType.maxiumOrderPrice`?

For `ShippingMethodType` maximum and minimum order price are returned correctly